test/client/before-test.js

```diff
@@ -1,4 +1,7 @@
 (function () {
+    $('script').addClass('script-TC2b9a6d');
+    $('link').addClass('ui-stylesheet-TC2b9a6d');
```

Comment:
I'm not sure if this correct, we make all scripts and links on page 'shadowed'. I think it's much better to explicitly specify which scripts and links we would like to hide.